src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
+import getToken from '../lib/tokens';
+import { Redirect } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 
 const Home = () => {
-  return <h1>Home</h1>;
+  // const [hasToken, setHasToken] = useState(false);
+
+  let history = useHistory();
+
+  const Token = function generateToken() {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    history.push('/list-view');
+
+    console.log(token);
+  };
+
+  // function fetchToken() {
+  //   localStorage.getItem('token');
+  //   history.push('/list-view');
+  // }
```

Comment:
Looks like we don't need this function anymore?

Reply:
Function removed! (●'◡'●)